apps/safe-claiming-app/src/components/steps/Claim/index.tsx

```diff
@@ -195,10 +195,12 @@ const Claim = ({ handleBack, state, handleUpdateState, handleNext }: Props) => {
             color: ({ palette }) => palette.secondary.main,
           }}
         />
-        <Typography variant="subtitle1">
+        <Typography variant="subtitle2">
           Awarded total allocation is{" "}
-          {formatAmount(Number(ethers.utils.formatEther(totalAllocation)), 2)}{" "}
-          SAFE
+          <span style={{ color: "#121312" }}>
```

Comment:
Could also use the theme color here e.g. `<Typography component="span" color="primary">`